src/Application/Operations/Bulk/BulkUpdateOperation.php

```diff
@@ -23,10 +23,10 @@ public static function from(iterable $iterable, string $indexName): self
     {
         $operation = new self($indexName);
 
-        if ($iterable instanceof \Traversable) {
-            $operation->models = iterator_to_array($iterable);
-        } else {
+        if (is_array($iterable)) {
             $operation->models = $iterable;
+        } else {
+            $operation->models = iterator_to_array($iterable);
```

Comment:
I added this change to ensure the PHP 8.2 mutation tests do not fail (as they do now on master)

Reply:
I am going to steal this for a new PR to fix master immediately :D